client/src/components/navbar.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import '../stylings/navbar.css';
+import RelatableLogo from '../images/relatable_full_logo.png';
+
+function Navbar() {
+  return (
+    <>
+      <nav className="navbar">
+        <div className="navbar-container">
+          <ul className="navbar-menu">
+            <li className="navbar-logo">
+              <img src={RelatableLogo} alt="Relatable Logo" id="logo" />
```

Comment:
Some margin-left on the logo would be good. Right now it is all the way left with no margin or padding, which looks bad